decidim-debates/lib/decidim/debates/feature.rb

```diff
@@ -13,11 +13,13 @@
 
   feature.settings(:global) do |settings|
     settings.attribute :comments_enabled, type: :boolean, default: true
+    settings.attribute :announcement, type: :text, translated: true, editor: true
   end
 
   feature.settings(:step) do |settings|
     settings.attribute :creation_enabled, type: :boolean, default: false
     settings.attribute :comments_blocked, type: :boolean, default: false
+    settings.attribute :announcement, type: :text, translated: true, editor: true
```

Comment:
You're missing the i18n string for these options

Reply:
No I don't! 🗡